### ISSUE TITLE
[Issue]: Jellyfin only finding 1 episode among entire show/season dumps

### ISSUE THREAD

**Issue by spoolr** [jellyfin/jellyfin#8023]
### Please describe your bug

Long time user.  I just moved 4 complete shows with every season into my media directory with Midnight Commander.  Of the 4, one worked correctly importing all episodes but the other three only found one episode from the dozens of files.   The shows are correctly labelled, Jellyfin does correctly provide metadata for the only episode it adds.  All the files are on the same physical drive, so it can't be a drive sleeping issue.  Forcing a rescan of the library doesn't find the new shows.  I had this occur the previous week and it took three separate media scans before finally finding them.  But aside from initiating the scan, I didn't change anything.  So far I've done multiple back to back scans and it's not find them.

For the scans, there is nothing in the logs error wise, just the usually virtual seasons stuff.  

I'm running Jellyfin in a docker.  It's the latest 0.8.  It likely updated around when these problems started.  But I'm also running Unraid which recently updated as well.

I've checked the Jellyfin drive which is an SSD for errors.  It's okay.  I've jumped into the docker and gone down to the media shares, and can see all the episodes listed.

Unable to resolve this, I renamed library.db and did a full scan over night and now most shows are now only showing one episode.  If I click on TV Shows, it shows the thumbnails with a 1 in the upper right corner indicated episode count (which is incorrect).  If I click the thumbnail to get details, it will only show 1 season, but on the S1E1 thumbnail it will show 40 or however many episodes in the corner of the thumbnail.  Metadata library just shows 1 episode, again correctly identified.  

It's very confusing that the first episode is correctly found and identified and something weird is happening to the rest.

### Jellyfin Version

10.8.0

### Environment

```markdown
- OS:Unraid
- Virtualization:Docker
- Clients:FireTV
- Browser:Brave
```

### Please attach any screenshots here

![Screenshot (92)](https://user-images.githubusercontent.com/3981555/175984882-1552c986-607d-4c8f-b744-24b56cc1a0f8.png)
![Screenshot (93)](https://user-images.githubusercontent.com/3981555/175984935-c207cd0d-c5dc-4470-8a06-d90a48f75b46.png)
![Screenshot (94)](https://user-images.githubusercontent.com/3981555/175984965-6a5a796d-624d-4683-ae12-24b8a9012f36.png)
![Screenshot (95)](https://user-images.githubusercontent.com/3981555/175984989-53dfd5d9-b61c-499d-b69e-58b7c053b3b8.png)
![Screenshot (96)](https://user-images.githubusercontent.com/3981555/175985019-c688b983-eca8-4cc7-a0ea-e246f787f2a1.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SenorSmartyPants** [comment]
This would be fixed in 10.8.1, released last night.

**Comment by thornbill** [comment]
Looks like a duplicate of https://github.com/jellyfin/jellyfin/issues/7855
